### PR TITLE
Fix untranslatable strings in admin

### LINKED ISSUE #4118: Admin - Untranslatable strings
## Description
The following strings are not wrapped in `intl.formatMessage` and therefore lack the ability to be translated.

- https://github.com/GCTC-NTGC/gc-digital-talent/blob/a68d302b9e97c7a40653bf521de1d02f0ef605ca/frontend/admin/src/js/components/dashboard/LatestRequestsTable.tsx#L65
- https://github.com/GCTC-NTGC/gc-digital-talent/blob/a68d302b9e97c7a40653bf521de1d02f0ef605ca/frontend/admin/src/js/components/searchRequest/SearchRequestTable.tsx#L51

## Screenshot

Screen Shot 2022-09-29 at 11 36 19

## Acceptance Criteria
- [ ] all strings wrapped in `intl.formatMessage` with a defaultMessage and description

### PR DESCRIPTION
**Pull request by JamesHuf**
Resolves #4118.

Sets the indicated strings in the issue to be translated in the future.

### CODE REVIEW

**Comment by petertgiles** [comment]
This is a live page.  Could you try to get the French translation in, too?  Maybe check with Matt or Gray or even email Nathan Golden?

File: frontend/admin/src/js/components/dashboard/LatestRequestsTable.tsx [hunk 1]
```diff
@@ -62,7 +62,12 @@ const LatestRequestsTable: React.FC<LatestRequestsTableProps> = ({ data }) => {
       accessor: ({ id, fullName }) =>
         tableViewItemButtonAccessor(
           paths.searchRequestView(id),
-          "request",
+          intl.formatMessage({
+            defaultMessage: "request",
+            id: "dKtPxj",
+            description:
+              "Text displayed on the search request table view buttons.",
```
Comment (mnigh) [inline frontend/admin/src/js/components/dashboard/LatestRequestsTable.tsx:69]: Consider changing this to "Text displayed after View text for Request table view action"

File: frontend/admin/src/js/components/searchRequest/SearchRequestTable.tsx [hunk 2]
```diff
@@ -48,7 +48,12 @@ export const SearchRequestTable: React.FunctionComponent<
         accessor: ({ id, fullName }) =>
           tableViewItemButtonAccessor(
             paths.searchRequestView(id),
-            "request",
+            intl.formatMessage({
+              defaultMessage: "request",
+              id: "dKtPxj",
+              description:
+                "Text displayed on the search request table view buttons.",
```
Comment (mnigh) [inline frontend/admin/src/js/components/searchRequest/SearchRequestTable.tsx:55]: Consider changing this to "Text displayed after View text for Search Request table view action"

**Comment by JamesHuf** [comment]
Ok, I've updated the descriptions and added the given translation.

**Review by mnigh — APPROVED** [review]
LGTM 👍 🇫🇷

**Pull request merged**